src/containers/save/_save.js

```diff
@@ -208,16 +218,16 @@ function buildSaveObject(action) {
             const saveType = 'page'
             const actionInfo = { cxt_ui: 'toolbar' }
             const showSavedIcon = true
-            const saveHash = md5(tab.url)
+            // const saveHash = md5(tab.url)
```

Comment:
Is there a reason we aren't removing the code completely?

Reply:
Good catch.  Fixed.